models/DataObject/ClassDefinition/Data/Multihref.php

```diff
@@ -721,7 +721,12 @@ public function preGetData($object, $params = [])
 
                 $setter = 'set' . ucfirst($this->getName());
                 if (method_exists($object, $setter)) {
-                    $object->$setter($data);
+                    $object->setObjectVar($this->getName(), $data);
+                    $this->markLazyloadedFieldAsLoaded($object);
+
+                    if (method_exists($object, "resetDirtyMap")) {
```

Comment:
could we work with an intereface here ... instead of method_exists?

Reply:
there is already an interface, namely DirtyIndicatorInterface, but wasn't use here